AzureMonitorAgent/packaging.sh

```diff
@@ -30,8 +30,7 @@ cp ../Common/WALinuxAgent-2.0.16/waagent .
 
 cp -r  ../LAD-AMA-Common/metrics_ext_utils .
 cp -r  ../LAD-AMA-Common/telegraf_utils .
-cp -r  ../Diagnostic/services .
-cp -f ../AzureMonitorAgent/services/metrics-extension.service services/metrics-extension.service
```

Comment:
why are we removing this?

Reply:
src and dst are the same, this is redundant